<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateData</class>
 <widget class="QDialog" name="UpdateData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="priceLE">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="clubContractL">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lastNameLE">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>131</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="idLE">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>41</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLE">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>71</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="footL">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leagueLE">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="surnameLE">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>101</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="ageSB">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>42</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="clubLE">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="contactEndL">
   <property name="geometry">
    <rect>
     <x>188</x>
     <y>60</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>contract end</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="numberSB">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>42</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="clubContractDE">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="footCB">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>72</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Left</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Right</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Both</string>
    </property>
   </item>
  </widget>
  <widget class="QDateEdit" name="contractEndDE">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ageL">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>age</string>
   </property>
  </widget>
  <widget class="QLabel" name="numberL">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>number</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updatePB">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
